subcommittees/policy-CoC/README.md

```diff
@@ -0,0 +1,33 @@
+# The Code of Conduct Policy Subcommittee
+
+## Purpose and Scope
+
+Enforce the Code of Conduct (CoC) in all parts of the Carpentries community
+
+## What We Do
+
+* communicate with the Carpentries community about the CoC, as well as reporting and enforcement guidelines
+* review and respond to reports of CoC violations as deemed appropriate by the situation
+
+## Contact
+
+* policy@carpentries.org
+* confidential@carpentries.org
+
+## Current Members
+
+* Pauline Barmby - chair (community)
+* Chris Hamm (community)
+* Simon Waldman (community)
+* Erin Becker (staff)
+* Jonah Duckles (staff)
```

Comment:
Thanks @k8hertweck. I think once the name of the committee is corrected this is ready to merge from my perspective.